CONTRIBUTING.md

```diff
@@ -1,3 +1,13 @@
+- [Contributing code](#contributing-code)
```

Comment:
Could you add a header before this text, e.g., "Table of Contents"?  It looks like something is missing without a title at the top.  

Or maybe an organization like this (without the bullets):

Contributing Code
- Table of Contents

General Guidelines for Contributing
-   Running the tests locally
-   Linting
-   General notes

Missing features ...

Reply:
Good point - I've added the header